firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/LiveContentResponse.kt

```diff
@@ -16,26 +16,47 @@
 
 package com.google.firebase.vertexai.type
 
-/* Represents the response from the server. */
+/**
+ * Represents the response from the server for live content updates. This class encapsulates the
+ * content data, the status of the response, and any function calls included in the response.
+ * @param data
+ */
 @PublicPreviewAPI
 public class LiveContentResponse
 internal constructor(
+
+  /** The main content data of the response. This can be null if there is no content. */
   public val data: Content?,
+
+  /**
+   * The status of the live content response. Indicates whether the response is normal, was
```

Comment:
I don't think it represents the status of the response, but more closely to the status of the conversation